test/e2e/clusterclaim_test.go

```diff
@@ -91,6 +91,43 @@ var _ = ginkgo.Describe("Testing ClusterClaim", func() {
 		}, eventuallyTimeout, eventuallyInterval).ShouldNot(gomega.HaveOccurred())
 	})
 
+	ginkgo.It("should not rollback the change of the ClusterClaim, if it's in consist list", func() {
```

Comment:
nit, CreateOnlyList ?

Reply:
Got, it I will fix it in the following up PR.

Reply:
Fixed